optaplanner-build-parent/pom.xml

```diff
@@ -1143,7 +1143,7 @@
               <groupId>org.apache.maven.plugins</groupId>
               <artifactId>maven-javadoc-plugin</artifactId>
               <configuration>
-                <source>${maven.compiler.source}</source>
+                <source>${maven.compiler.release}</source>
```

Comment:
This probably picked up the property from jboss-parent, so 8 instead of 11 as it should.